package.json

```diff
@@ -24,56 +24,56 @@
   },
   "dependencies": {
     "async": "0.2.7",
-    "aws-sdk": "^2.2.4",
+    "aws-sdk": "^2.23.0",
     "bestlang": "0.0.1",
-    "body-parser": "^1.13.1",
-    "chokidar": "^1.4.2",
+    "body-parser": "^1.17.1",
+    "chokidar": "^1.6.1",
     "colors": "^1.1.2",
-    "compression": "^1.5.0",
-    "cookie-parser": "^1.3.5",
+    "compression": "^1.6.2",
+    "cookie-parser": "^1.4.3",
     "cookie-session": "^1.1.0",
     "cookies-js": "~1.2.1",
-    "cors": "^2.7.1",
-    "cryptr": "^1.0.0",
-    "csurf": "^1.8.3",
-    "express": "^4.13.0",
+    "cors": "^2.8.1",
+    "cryptr": "^2.0.0",
+    "csurf": "^1.9.0",
+    "express": "^4.15.2",
     "form-data": "^0.2.0",
-    "fs-writefile-promise": "1.0.3",
-    "glob": "^6.0.4",
+    "fs-writefile-promise": "2.0.0",
+    "glob": "^7.1.1",
     "grunt": "~0.4.1",
-    "grunt-cli": "^0.1.13",
-    "grunt-contrib-csslint": "0.1.2",
+    "grunt-cli": "^1.2.0",
+    "grunt-contrib-csslint": "2.0.0",
     "grunt-contrib-jshint": "^0.11.2",
-    "grunt-contrib-less": "^1.0.1",
-    "grunt-contrib-requirejs": "^0.4.4",
-    "grunt-execute": "0.1.5",
+    "grunt-contrib-less": "^1.4.0",
+    "grunt-contrib-requirejs": "^1.0.0",
+    "grunt-execute": "0.2.2",
     "grunt-git": "^0.3.4",
-    "grunt-lesslint": "^1.1.14",
+    "grunt-lesslint": "^3.2.0",
     "grunt-npm": "git://github.com/sedge/grunt-npm.git#branchcheck",
     "grunt-update-submodules": "^0.4.1",
     "habitat": "3.1.2",
-    "helmet": "2.1.2",
-    "jit-grunt": "^0.9.1",
-    "jquery": "~2.1.4",
-    "less-middleware": "2.0.1",
+    "helmet": "3.4.1",
+    "jit-grunt": "^0.10.0",
+    "jquery": "~3.1.1",
+    "less-middleware": "2.2.0",
     "memorystream": "^0.3.1",
-    "moment": "~2.11.2",
-    "morgan": "^1.7.0",
-    "multer": "^1.0.1",
-    "node-uuid": "~1.4.3",
-    "npm-run-all": "^1.5.1",
-    "nunjucks": "^2.4.3",
+    "moment": "~2.17.1",
+    "morgan": "^1.8.1",
+    "multer": "^1.3.0",
+    "node-uuid": "~1.4.7",
+    "npm-run-all": "^4.0.2",
+    "nunjucks": "^2.3.0",
     "properties-parser": "0.3.1",
     "q-io": "1.13.2",
-    "request": "2.21.0",
-    "serve-favicon": "^2.3.0",
-    "tar-stream": "^1.2.1",
+    "request": "2.80.0",
+    "serve-favicon": "^2.4.1",
+    "tar-stream": "^1.5.2",
     "throng": "^4.0.0",
-    "time-grunt": "^1.2.1",
-    "uuid": "^2.0.1",
-    "webmaker-analytics": "~0.1.5",
+    "time-grunt": "^1.4.0",
+    "uuid": "2.0.3",
+    "webmaker-analytics": "~0.1.7",
     "webmaker-i18n": "^0.3.31",
-    "wolfy87-eventemitter": "~4.3.0"
+    "wolfy87-eventemitter": "~5.1.0"
```

Comment:
I have no idea what this is used for, which makes it all the more important that we don't bump this up a major version number outright - let's file a separate issue for this dependency, too.